src/callbacks/callbacks.jl

```diff
@@ -320,7 +320,7 @@ function save_to_disk_func(integrator)
 
     if p.atmos.turbconv_model isa EDMFX
         (; ᶜspecific⁰, ᶜu⁰, ᶜts⁰) = p
-        (; ᶜu⁺, ᶜts⁺, ᶜa⁺, ᶜa⁰) = diagnostic_sgs_quantities(Y, p, t)
+        (; ᶜu⁺, ᶜts⁺, ᶜa⁺, ᶜa⁰) = output_sgs_quantities(Y, p, t)
```

Comment:
I like `output_sgs_quantities`!

Reply:
Also, could you remind me what the \^+ is symbolizing here?

Reply:
It's for bulk draft properties (which I am setting to the first and only updraft for diagnostic edmf:)